provider/internal/device/asa/resource.go

```diff
@@ -124,6 +126,9 @@ func (r *AsaDeviceResource) Schema(ctx context.Context, req resource.SchemaReque
 				Validators: []validator.List{
 					listvalidator.UniqueValues(),
 				},
+				PlanModifiers: []planmodifier.List{ // TODO: use set when we go to 1.0.0
```

Comment:
why not use set from now?

Reply:
Because I do not want to break existing code. I want to keep the breaking change when we want to upgrade to new major version, it is a temporary fix for now.

Reply:
Could you please link to a JIRA ticket to switch this over?

Reply:
What would the backwards incompatible change involve? We have a lot of code already, internally as well as for our labs, using this stuff.

Reply:
Yes done, also see https://jira-eng-rtp3.cisco.com/jira/browse/LH-71968 for the backwards incompatible changes that may involve.